tests/block/k8s/create_block_test.go

```diff
@@ -57,7 +60,12 @@ func (s *K8sBlockImageCreateSuite) SetupSuite() {
 
 	var err error
 
-	s.testClient, err = clients.CreateTestClient(tests.Platform)
+	rook, err := installer.NewK8sRookhelper()
```

Comment:
rook context is not used anywhere in this test, but I was thinking about other tests where you kill pods and have to install something again, 
I know are using oo patterns where they are not needed, we want to clean up that as part of #774,
Now that I think about it, I  feel like installer can be static.